tests/functional/spec/radio.spec.js

```diff
@@ -53,22 +53,94 @@ describe('Radio button with "other" option', function() {
+  it('Given I have previously added text in other textfield and saved, when I change the answer to a different answer, then the text entered in other field must be wiped.', function() {
     //Given
     openQuestionnaire(radio_schema);
 
     //When
     RadioMandatoryPage.clickOther().setOtherInputField('Other Text').submit()
-    RadioNonMandatory.clickTopPrevious()
+    RadioNonMandatoryPage.clickTopPrevious()
     RadioMandatoryPage.clickRadioMandatoryAnswerBacon().submit()
-    RadioNonMandatory.clickTopPrevious()
+    RadioNonMandatoryPage.clickTopPrevious()
 
     //Then
     RadioMandatoryPage.clickOther()
     expect(RadioMandatoryPage.getOtherInputField()).to.equal('')
   })
 
+  it('Given I have previously selected the None option and saved, When I return to the screen, Then my choice is remembered.', function() {
+    // https://github.com/ONSdigital/eq-survey-runner/issues/1013
+    // Given
+    openQuestionnaire(radio_schema);
+
+    expect(RadioMandatoryPage.RadioMandatoryAnswerNoneIsSelected()).to.be.false
+    RadioMandatoryPage.clickRadioMandatoryAnswerNone()
+    expect(RadioMandatoryPage.RadioMandatoryAnswerNoneIsSelected()).to.be.true
+    RadioMandatoryPage.submit()
+
+    // When
+    RadioNonMandatoryPage.clickTopPrevious()
+
+    // Then
+    expect(RadioMandatoryPage.RadioMandatoryAnswerNoneIsSelected()).to.be.true
+  })
+
+  it('Given I have previously selected an option and saved, When I return to the screen and change my answer twice, Then my new choice is remembered and is shown on the Summary screen.', function() {
+    // Given
+    openQuestionnaire(radio_schema);
+
+    // Nothing should be selected by default
+    expect(RadioMandatoryPage.RadioMandatoryAnswerNoneIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerBaconIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerEggsIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerSausageIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerOtherIsSelected()).to.be.false
+
+    // choose Eggs first
+    RadioMandatoryPage.clickRadioMandatoryAnswerEggs()
+    expect(RadioMandatoryPage.RadioMandatoryAnswerEggsIsSelected()).to.be.true
+    RadioMandatoryPage.submit()
+
+    // When I return to the screen
+    RadioNonMandatoryPage.clickTopPrevious()
+
+    // Then my choice is remembered (Eggs)
+    expect(RadioMandatoryPage.RadioMandatoryAnswerNoneIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerBaconIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerEggsIsSelected()).to.be.true
+    expect(RadioMandatoryPage.RadioMandatoryAnswerSausageIsSelected()).to.be.false
+    expect(RadioMandatoryPage.RadioMandatoryAnswerOtherIsSelected()).to.be.false
```

Comment:
We could do with a set of common scenarios for all question types; such as checking default value on first page load; submitting and returning to screen to see value persisted, changing the value, removing an answer etc. At the moment coverage of these typical use cases is patchy. If WTForms does change this issue could get re-introduced so perhaps safer to test our usage of it until we have sufficient use cases covered generically?